packages/one-app-dev-bundler/esbuild/plugins/one-app-index-loader-injectors/holocron-module-register-injector.js

```diff
@@ -27,7 +27,7 @@ export default class HolocronModuleRegisterInjector {
       return content;
     }
 
-    return `${content};
```

Comment:
@Matthew-Mallimo  re https://github.com/americanexpress/one-app-cli/pull/536#discussion_r1297794157
Each injector would add an additional `;` to the content which should already be terminated with a semi colon.